lib/demo/interactors/create_comment.rb

```diff
@@ -0,0 +1,23 @@
+require 'lotus/interactor'
+
+module Demo
+  class CreateComment
+    include Lotus::Interactor
+
+    def initialize(params:, repository: CommentRepository)
+      @repository = repository
+      @params = params
+    end
+
+    def call
```

Comment:
The mutation of the state that I suggest with `persist_comment` has a specific reason. All the instance variables of an interactor are available as accessors of the `Lotus::Interactor::Result` instance returned by `#call`.

In this case:

```ruby
result = CreateComment.new(params).call
result.comment
```

This is useful to pass a payload to the view.